### PR TITLE
Log HTTP requests for easier debugging

### PR DESCRIPTION
**Pull request by jasontyu**
Log all HTTP requests in express

> Screen Shot 2020-06-28 at 9 02 25 PM
> (Caption: Console logs when I visit `localhost:8080/test-db`)

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-var express = require('express')
-var router = express.Router()
-var db = require('../db/index.js')
+const express = require('express')
+const router = express.Router()
+const db = require('../db')
```
Comment (LeekSwan) [inline routes/index.js:3]: [nit] Does /index.js not need to be specified? There are currently 2 files in db folder.

File: start.js [hunk 2]
```diff
@@ -11,16 +11,19 @@ if (process.env.NODE_ENV === 'dev') {
 
 const express = require('express')
 const path = require('path');
+const morgan = require('morgan')
```
Comment (LeekSwan) [inline start.js:14]: [nit] are we using semicolons or nah

Comment (jasontyu) [reply start.js:14]: uhhh I think no semicolons

I kind of automatically add semicolons, but this article explains why it's not that useful to: https://feross.org/never-use-semicolons/

Good catch!

File: routes/index.js (hunk 1 above)
Comment (jasontyu) [reply routes/index.js:3]: I think this kinda explains it? https://www.reddit.com/r/javascript/comments/7rv221/to_indexjs_or_not_to_indexjs/

File: start.js (hunk 2 above)
Comment (jasontyu) [reply start.js:14]: We should add a linter to catch/auto-fix these small things

**Pull request merged**